File: Challenges/C++/ex0.cpp
#include<iostream>

using namespace std;

int main(int argc, char** argv){
	if(argc!=2){
		cout << "Usage: ./prog input\n";
		cout << "Hint: ./prog -h\n";
	}else if(string(argv[1]) == "-h"){
		cout << "The desired input is in plain text.\n";
	}else{
		string(argv[1])=="password"?cout << "Score!\n":cout << "Fail!\n";
	}
	return 0;
}

File: Challenges/Go/ex0.go
package main

import "os"
import "fmt"

func main(){
	if len(os.Args) != 2{
		fmt.Println("Usage: ./prog input")
		fmt.Println("Hint: ./prog -h")
	}else if os.Args[1] == "-h"{
		fmt.Println("The input you seek is hardcoded.")
	}else{
		if os.Args[1] == "password"{
			fmt.Println("Score!")
		}else{
			fmt.Println("Fail!")
		}
	}
}
